doc/zh-cn/plugins/limit-count.md

```diff
@@ -26,17 +26,18 @@
-| rejected_code  | integer | 可选         | 503     | [200,600]                                                                | 当请求超过阈值被拒绝时，返回的 HTTP 状态码                                                                                                                                                  |
-| policy         | string  | 可选         | "local" | ["local", "redis"]                                                       | 用于检索和增加限制的速率限制策略。可选的值有：`local`(计数器被以内存方式保存在节点本地，默认选项) 和 `redis`(计数器保存在 Redis 服务节点上，从而可以跨节点共享结果，通常用它来完成全局限速) |
-| redis_host     | string  | `redis` 必须 |         |                                                                          | 当使用 `redis` 限速策略时，该属性是 Redis 服务节点的地址。                                                                                                                                  |
-| redis_port     | integer | 可选         | 6379    | [1,...]                                                                  | 当使用 `redis` 限速策略时，该属性是 Redis 服务节点的端口                                                                                                                                    |
-| redis_password | string  | 可选         |         |                                                                          | 当使用 `redis` 限速策略时，该属性是 Redis 服务节点的密码。                                                                                                                                  |
-| redis_timeout  | integer | 可选         | 1000    | [1,...]                                                                  | 当使用 `redis` 限速策略时，该属性是 Redis 服务节点以毫秒为单位的超时时间                                                                                                                    |
+| 名称            | 类型     | 必选项       | 默认值  | 有效值                                                       | 描述                                                         |
+| --------------- | -------- | ------------ | ------- | ------------------------------------------------------------ | ------------------------------------------------------------ |
+| count           | integer  | 必须         |         | [0,...]                                                      | 指定时间窗口内的请求数量阈值                                 |
+| time_window     | integer  | 必须         |         | [0,...]                                                      | 时间窗口的大小（以秒为单位），超过这个时间就会重置           |
+| key             | string   | 必须         |         | ["remote_addr", "server_addr", "http_x_real_ip", "http_x_forwarded_for"] | 用来做请求计数的依据                                         |
+| rejected_code   | integer  | 可选         | 503     | [200,600]                                                    | 当请求超过阈值被拒绝时，返回的 HTTP 状态码                   |
```

Comment:
why change those lines?

Reply:
Should be formatted by the editor, the content has not changed.